pkg/daemon/sync.go

```diff
@@ -149,6 +153,7 @@ func doSync(ctx context.Context, manifestsStore manifests.Store, clus cluster.Cl
 	logger log.Logger) (map[string]resource.Resource, []event.ResourceError, error) {
 	resources, err := manifestsStore.GetAllResourcesByID(ctx)
 	if err != nil {
+		updateSyncManifestsMetric(0, 1)
```

Comment:
As above

Reply:
Probably this one is a single place where using metrics still makes sense but it must be adjusted to include number of manifests.
This is because `flux_daemon_sync_duration_seconds_count` usually say something is wrong with infrastructure and thus devops intervention is required. `flux_daemon_sync_manifests` on the other hand is info about invalid manifests in git.
The problem is in either case we can't say how many manifest were succeeded here because effectively success number is 0.

So @squaremo is it ok to leave metric here and put number of manifest parsing errors into it? Or just remove the line?

Reply:
I get what you mean -- it _is_ a problem with individual manifests, we just don't have the numbers handy. How about this: remove it from here for now, and we can consider a further change after this is merged: something like expanding this metric to have a label for the particular stage at which something failed; or, having a more fine-grained metrics for the duration (and success/failure) of sync phases generation / parsing / validation / application / gc.